onnxruntime/python/tools/transformers/convert_generation.py

```diff
@@ -182,12 +182,21 @@ def parse_arguments(argv: Optional[List[str]] = None) -> argparse.Namespace:
     )
     output_group.set_defaults(pad_vocab_size=True)
 
+    output_group.add_argument(
+        "-sgd",
+        "--separate_gpt2_decoder_for_init_run",
+        required=False,
+        action="store_true",
+        help="Have separate decoder subgraphs for initial and remianing runs. This allows for optimizations based on sequence lengths in each subgraph",
+    )
+    output_group.set_defaults(separate_gpt2_decoder_for_init_run=False)
```

Comment:
Default will be made True after kernel support is added and the change is validated